ecosystem/typescript/sdk/src/generated/core/request.ts

```diff
@@ -401,7 +401,14 @@ export const request = <T>(config: OpenAPIConfig, options: ApiRequestOptions): C
 
                 catchErrorCodes(options, result);
 
-                resolve(result.body);
+                // Attach the response headers to the output. This is a hack to fix
+                // https://github.com/ferdikoomen/openapi-typescript-codegen/issues/1295
+                const out = result.body;
+                try {
+                    out["__headers"] = response.headers;
```

Comment:
Would response always have headers besides `x-aptos-cursor`? e.g. cookie headers. If so, we introduced byproduct "__headers" to the endpoints other than get resources and modules.

Reply:
Yes it'll have all the information from the other headers too. In some ways this is sort of a feature. The information is there, we just throw it out right now. Saying that, I'm not sure we'd want people to depend on this value as it is now, it's a bit of a hack. Still, it has merit, this way you could avoid some queries.